content/05.administrative.md

```diff
@@ -32,7 +32,7 @@ AJW is a member of the Scientific Advisory boards for Alexion and DayOne Biophar
 ## Figure Titles and Legends
```

Comment:
Generally, I think we have changes to figure legends happening in two separate places and some of them are now consistent as a consequence?